scheduler/env.c

```diff
@@ -261,8 +261,9 @@ find_env(const char *name)		/* I - Variable name */
   int		i;			/* Looping var */
   size_t	namelen;		/* Length of name */
 
+  namelen = strlen(name);
```

Comment:
Two blank lines after declarations at the first level under a function. But why move the initializer out of the for loop?